src/objects/zcl_abapgit_object_devc.clas.abap

```diff
@@ -354,45 +354,8 @@ CLASS zcl_abapgit_object_devc IMPLEMENTATION.
         RETURN.
       ENDIF.
 
-      TRY.
-          CALL METHOD li_package->('SET_CHANGEABLE')
-            EXPORTING
-              i_changeable                = abap_true
-              i_suppress_dialog           = abap_true " Parameter missing in 702
```

Comment:
`suppress_dialog` sounds like something we would like to do, the `set_lock` method doesn't set the parameter

Reply:
yes, it should have been in set_lock all along